### PR TITLE
Example Usage

### PR DESCRIPTION
**Pull request by Noah-Huppert**
**Why?**  

I started using the basictracer in my application b/c I know I want to use the Open Tracing standard but I'm not sure which logging provider I want to use.  

In order to get started integrating the Open Tracing standard into my application I started using basictracer to make spans.  

I thought it would be useful to add this code to the README in case others want to do the same.

### CODE REVIEW

File: examples/logging.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        for log in span.logs:
+            span_logger.debug(log.key_values)
+
+logger = logging.getLogger(__name__)
+recorder = LogSpanRecorder(logger)
+
+tracer = BasicTracer(recorder=recorder)
+tracer.register_required_propagators()
+
```
Comment (yurishkuro) [inline examples/logging.py]: maybe add a main() function and a few comments explaining how to run it (or add a README to the dir).

Comment (Noah-Huppert) [reply examples/logging.py]: Good idea! I'll add an example span as well.

File: examples/span_logging.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 1}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 2}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.01e+00 S, {'i': 3}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 4}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 5}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 6}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 7}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 8}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 9}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.main[1.00e+01 S]: finished
```
Comment (yurishkuro) [inline examples/span_logging.py]: I am not sure why this format of logging the span is particularly useful. It doesn't capture span IDs so you cannot see causality.

Comment (Noah-Huppert) [reply examples/span_logging.py]: This is true, I'll take a crack at making this more useful and pretty.

**Comment by Noah-Huppert** [comment]
Made new logging format:

```
[DEBUG   ] span_logging.loop.18221754867226935446[1.00 S i=0]: message=Sleeping for 1 second
```

Thanks for all the feedback you've been giving me. I really appreciate your help getting this PR ready.

**Comment by Noah-Huppert** [comment]
oops I was grabbing the wrong Id, gotta fix that

**Comment by carlosalberto** [comment]
@yurishkuro think this could go in? Looks fine by me ;)

**Pull request merged**